content/announcements/v5.0/v5.0.0.md

```
@@ -0,0 +1,126 @@
+---
+title: 'v5.0.0'
+description: 'Next generation Telemetry Agent for Logs, Metrics and Traces.'
+url: "/announcements/v5.0.0/"
+release_date: 2026-03-24
+publishdate: 2026-03-24
+ver: v5.0.0
+herobg: "/images/hero@2x.jpg"
+latestVer: true
+---
+
+###### KNOWLEDGE BASE
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h6

(MD001, heading-increment)